### ISSUE TITLE
Could not resolve "json-formatter-js"

### ISSUE THREAD

**Issue by Gemorroj** [mohsen1/json-formatter-js#134]
package manager: yarn 1
framework: angular 17

after upgrade 2.3.4 to 2.5.9
```
✘ [ERROR] Could not resolve "json-formatter-js"
    src/app/my.component.ts:3:26:
      3 │ import JSONFormatter from 'json-formatter-js';
        ╵                           ~~~~~~~~~~~~~~~~~~~
  The package target "dist/json-formatter.mjs" is invalid because it doesn't start with "./":
    node_modules/json-formatter-js/package.json:11:16:
      11 │       "import": "dist/json-formatter.mjs",
         ╵                 ~~~~~~~~~~~~~~~~~~~~~~~~~
  You can mark the path "json-formatter-js" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
```

**Comment by sonallux** [comment]
@Gemorroj this bug should be fixed in version `2.5.10`

**Comment by Gemorroj** [comment]
@sonallux yes, it works. thank you
